public/css/main.css

```diff
@@ -0,0 +1,883 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
can we not check in built css? imo files created by the build process make diffs harder to read and aren't necessary to be checked in. if you agree, we can just add this file to the `.gitignore`